lib/containers/__tests__/artist-tests.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+'use strict'
+import 'react-native'
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import Artist from '../artist'
+
+jest.mock('../../components/artist/shows', () => 'Shows')
+jest.mock('../../components/artist/artworks', () => 'Artworks')
+jest.mock('../../components/artist/header.js', () => 'Header')
+jest.mock('../../components/artist/about.js', () => 'About')
+
+jest.mock('../../components/spinner.js', () => 'ARSpinner')
+jest.mock('../../components/opaque_image_view.js', () => 'AROpaqueImageView')
+jest.mock('../../components/switch_view.js', () => 'ARSwitchView')
+
+describe('availableTabs', () => {
+  it('returns nothing if artist has no metadata, shows, or works', () => {
+    const artist = new Artist()
+    artist.props = artistProps(false)
```

Comment:
This `artistProps` method can be found down [here](https://github.com/artsy/emission/pull/414/files#diff-39671116ef994b68054be6fca954e02dR94)